modules/ROOT/pages/clustering/clustering-advanced/multi-data-center-routing.adoc

```diff
@@ -1,5 +1,5 @@
 :description: This section shows how to configure Neo4j servers so that they are topology/data center-aware. It describes the precise configuration needed to achieve a scalable multi-data center deployment.
-:page-aliases: clustering-advanced/multi-data-center/index.adoc
+:page-aliases: clustering-advanced/multi-data-center/index.adoc, clustering-advanced/multi-data-center/load-balancing.adoc
```

Comment:
```suggestion
:page-aliases: clustering-advanced/multi-data-center/index.adoc, clustering-advanced/multi-data-center/load-balancing.adoc, clustering-advanced/multi-data-center/configuration.adoc
```

Reply:
There is one more page in this section without a page-alias  (https://neo4j.com/docs/operations-manual/4.4/clustering-advanced/multi-data-center/configuration/). I think we should add a page-alias for it. 

One more page (https://neo4j.com/docs/operations-manual/4.4/clustering-advanced/multi-data-center/design/) can be left without a page-alias because this content is relevant only for 4.4.